Resources/Prototypes/Entities/Tiles/liquid_plasma.yml

```
@@ -92,4 +94,6 @@
     # Goobstation - fishing
   - type: Tag
     tags:
-    - HideContextMenu
\ No newline at end of file
+    - HideContextMenu
+  - type: CosmicCorruptible # DeltaV - Cosmic Cult
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)